src/System.Private.CoreLib/shared/System/Environment.Unix.cs

```diff
@@ -158,19 +153,25 @@ private static string GetFolderPathCoreWithoutValidation(SpecialFolder folder)
                 case SpecialFolder.MyVideos:
                     return ReadXdgDirectory(home, "XDG_VIDEOS_DIR", "Videos");
 
+#if PLATFORM_OSX
                 case SpecialFolder.MyMusic:
-                    return s_isMac ? Path.Combine(home, "Music") : ReadXdgDirectory(home, "XDG_MUSIC_DIR", "Music");
+                    return Path.Combine(home, "Music");
                 case SpecialFolder.MyPictures:
-                    return s_isMac ? Path.Combine(home, "Pictures") : ReadXdgDirectory(home, "XDG_PICTURES_DIR", "Pictures");
+                    return Path.Combine(home, "Pictures");
                 case SpecialFolder.Fonts:
-                    return s_isMac ? Path.Combine(home, "Library", "Fonts") : Path.Combine(home, ".fonts");
-
+                    return Path.Combine(home, "Library", "Fonts");
                 case SpecialFolder.Favorites:
-                    if (s_isMac) return Path.Combine(home, "Library", "Favorites");
-                    break;
+                    return Path.Combine(home, "Library", "Favorites");
                 case SpecialFolder.InternetCache:
-                    if (s_isMac) return Path.Combine(home, "Library", "Caches");
-                    break;
+                    return Path.Combine(home, "Library", "Caches");
+#else
+                case SpecialFolder.MyMusic:
```

Comment:
where did this change come from?

Reply:
I don't remember having pushed/changed this file. I will push force again without this file.

Reply:
It's done, sorry!